### ISSUE TITLE
ForbiddenComments don't report TODO: in KDoc

### ISSUE THREAD

**Issue by alwaystest** [detekt/detekt#3273]
## Expected Behavior

`TODO:` in KDoc should be reported.

## Observed Behavior

`TODO:` in KDoc not be reported.

## Steps to Reproduce

```kotlin
/**
 * TODO: this is not reported
 */
object Test {

    fun testFunc() {
        /**
         * TODO: this is not reported, too.
         */
    }
}
```

`detekt -r xml:result.xml -i Test.kt`

Nothing is reported in result.xml

## Your Environment

* Version of detekt used:  1.14.2 CLI
* Version of Gradle used (if applicable):
* Operating System and version: OSX 10.15.4
* Link to your project (if it's a public repository):

**Comment by schalkms** [comment]
Thanks for reporting this!
This is reproducible.

**Comment by schalkms** [comment]
Fix is on it' way and will be merged for the next detekt release.
